server/controller/FlightController.js

```diff
@@ -591,14 +592,17 @@ async function runChecksStartCalculationsStoreFixes(
   if (!skipAllChecks && !skipMidflightCheck)
     detectMidFlightIgcStart(takeoff, fixes);
   await service.checkIfFlightWasNotUploadedBefore(flightDbObject);
-  await service.storeFixesAndAddStats(flightDbObject, fixes);
+  const fixesDbObject = await service.storeFixesAndAddStats(
+    flightDbObject,
+    fixes
+  );
 
-  if (!skipAllChecks) {
```

Comment:
Why did this line get removed? (skipallchecks)

Reply:
Because I didn't saw any reason for it to be there.
METAR data has nothing to do w any checks.

Reply:
Yes, but the intention was that an admin can upload a flight even if there are problems with the METAR Api.
This is debatable (but maybe better in another PR to keep topics separated?)

Reply:
agree